src/redux/products/products.sagas.js

```diff
@@ -65,4 +190,5 @@ export function* handleProductsErrors(e) {
 
 export default function* productsSaga() {
   yield takeEvery(GET_ALL_PRODUCTS, handleGetAllProducts);
+  yield takeEvery(GET_FILTRED_PRODUCTS, handleFilterLoad);
```

Comment:
I don't think that we need to handlers for fetching products, it is enough to have one `GET_PRODUCTS` that will use the filter for fetching products

Reply:
fixed